client/src/views/workspaces/WorkspaceSharingModal.vue

```diff
@@ -3,18 +3,51 @@
 <template>
   <ion-page class="modal">
     <ms-modal
-      :title="'WorkspaceSharing.title'"
+      :title="workspaceName"
```

Comment:
This will create an error since title is expecting a `Translatable`. It will pass the workspaceName to `$msTranslate` but there won't be any key for it. You should use `I18n.valueAsTranslatable` in this case (https://github.com/Scille/megashark-lib/blob/cff992d182818775dc4084185c2854e8435060d5/lib/services/translation.ts#L173)